### ISSUE TITLE
region not mapping across different profiles

### ISSUE THREAD

**Issue by varunchandak** [wallix/awless#182]
Suppose I have: 
* profile A with region X
* profile B with region Y

Using `awless -p A config set aws.region X`, whenever I try to use profile B, it uses the region X.

The correct working should be `A::X` and `B::Y`, not `B::X`

**Comment by fxaguessy** [comment]
That's an interesting feedback. For now, in `awless` we chose to manage independently regions and profiles, but it could also make sense to change both region and profile when switching profile.

Note anyway that you can use `awless switch` to both region and profile: e.g. `awless sw A us-west-1`.

We will discuss about this feature and its consequences on UX, for example: 
- do we want to change region transparently when doing `awless -p A ...`
- only when using `awless switch` command ?

**Comment by varunchandak** [comment]
Never tried the `awless switch` but I'd recommend to use `awless -p A ...` as that's how we call the profiles.

**Comment by awilkins** [comment]
Since you can declare `region` in the `.aws/` config files and the aws cli tools respect this as the default region for that profile, it would be "least surpriseful" if `awless` did the same thing and switched to the region associated with the profile in the AWS CLI config when switching to that config.

**Comment by simcap** [comment]
@awilkins I agree with you. We want to not least suprise the user and follow usual loading of the aws config. We also want to make clear to the user what is going one.

We are in the process of reviewing that, making sure it is proper and it should be included in the next release. 

Thanks for the feedback!

**Comment by simcap** [comment]
I am renaming this issue to reflect the actual bug

**Comment by simcap** [comment]
@varunchandak if you have time you can use the latest master (stable and compatible) and let me know if that fixes the issue. Cheers!

**Comment by varunchandak** [comment]
Sure...!

**Comment by simcap** [comment]
... use `awless switch ....` to update region and/or profile as `awless config set` is deprecated for region/profile
